src/pdf/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -89,14 +90,11 @@ interface LabelCardProps {
     quantity: number;
 }
 
-const convertDateFormat = (date: string): string => {
-    const dateArray = date.split("-");
-    return `${dateArray[2]}/${dateArray[1]}/${dateArray[0]}`;
-};
-
 const SingleLabelCard: React.FC<LabelCardProps> = ({ data, index, quantity }) => {
-    return (
-        data.full_name !== "Deleted Client" && (
+    if (data.full_name === "Deleted Client") {
+        return null;
+    } else {
+        return (
```

Comment:
Sorry, I might not have been clear. `SingleLabelCard` should always return JSX, no conditionals or null returns. Where it's called (`ShippingLabelForSingleParcel`) we should filter the array we map across using this conditional.